test/suites/client-LocalSettings.php

```diff
@@ -57,32 +31,3 @@
 $wgWBClientSettings['siteLinkGroups'] = [ 'mywikigroup' ];
 $wgWBClientSettings['siteGlobalID'] = 'client_wiki';
 
-foreach (glob("LocalSettings.d/*.php") as $filename)
```

Comment:
Yay for making this all go away! So this list was comprehensive excluding our added extensions before, and it's ok that no extensions are being added now?